src/DegenGambit.sol

```diff
@@ -601,6 +601,23 @@ contract DegenGambit is ERC20, ReentrancyGuard {
         }
     }
 
+    // Payout Estimate function to easily display current payouts estimate at time of function call
+    function payoutEstimate()
```

Comment:
This name is a bit confusing.

We are using `payout` to refer to the outcome of a single reel configuration (with `left`, `center`, and `right` inputs).

Maybe we should call this function `prizeEstimates`?

Or just `prizes` would be better IMO.

Reply:
Yes, I'll make that change.